src/lib/site-import/status.ts

```diff
@@ -198,7 +220,9 @@ export async function importSqlCheckStatus( {
 		let statusMessage;
 		switch ( overallStatus ) {
 			case 'success':
-				statusMessage = `Success ${ sprite } imported data should be visible on your site ${ env.primaryDomain.name }.`;
+				statusMessage = `Success ${ sprite } imported data should be visible on your site ${
+					env.primaryDomain?.name ?? 'N/A'
+				}.`;
```

Comment:
These things are to keep ESLint happy: our ruleset does not alow `undefined` as a template parameter.